torch/_inductor/lowering.py

```diff
@@ -2611,6 +2611,8 @@ def inner(
         dtype = dtype or torch.get_default_dtype()
         if len(size) == 1 and isinstance(size[0], (list, tuple, torch.Size)):
             size = tuple(size[0])
+        for s in size:
+            assert not isinstance(s, torch.SymInt)
```

Comment:
This makes the later problem trigger more clearly @eellison

Reply:
Maybe add a comment or update assertion along the lines of `should be Symbol not SymInt`